index.bs

```diff
@@ -1036,6 +1037,10 @@ An <dfn>attribution rate-limit record</dfn> is a [=struct=] with the following i
 :: A [=moment=].
 : <dfn>expiry time</dfn>
 :: Null or a [=moment=].
+: <dfn>entity ID</dfn>
+:: An [=event-level report=]'s [=event-level report/report ID=] or an
+    [=aggregatable report=]'s [=aggregatable report/report ID=] or an
+    [=attribution source=]'s [=attribution source/source identifier=].
```

Comment:
Where is this set for the source?

Reply:
I have added the set line for this, but it is not used in querying anywhere yet.